### DISCUSSION TITLE
[Threat] A Trojan?

### DISCUSSION THREAD

**Discussion by Golgovskiy** [TagStudioDev/TagStudio#510] [General]
The https://github.com/TagStudioDev/TagStudio/releases/download/v9.4.1/tagstudio_windows_x86_64.zip file is detected as a Trojan:Python/ZkarletFlash. VirusTotal also has some detections.
On Virustotal analysis, it seems to be doing something with TeamViewer files? Probably for remote control.

**Comment by CyanVoxel** [comment]
See the other discussions and issues on this:
- https://github.com/TagStudioDev/TagStudio/discussions/294
- https://github.com/TagStudioDev/TagStudio/issues/276
- https://github.com/TagStudioDev/TagStudio/issues/361
- https://github.com/TagStudioDev/TagStudio/issues/420

TLDR: These are false positives which are common with PyInstaller-bundled Python applications.

**Reply by yedpodtrzitko** [reply]
Since this topic is being brought up repeatedly, it could be worth pinning the topic here in Discussions, and perhaps in Issues as well.
